rocketpy/rocket/aero_surface/fins/fins.py

```diff
@@ -373,13 +373,9 @@ def compute_forces_and_moments(
         stream_mach,
         rho,
         cp,
-        _,
-        omega1,
-        omega2,
-        omega3,
+        omega,
```

Comment:
I would rename it to `w`.

Btw the docstring of this function is incomplete. We should either remove the papameters section or complete it

Reply:
Line 1513 of the `Flight` class sets `w = Vector([omega1, omega2, omega3])`. It seems to me that `omega` is a better name, so perhaps a better choice is to rename `w` to `omega`?

Reply:
I think `w` is used in our paper, but the user should really be reading "omega" instead of "double u".

I agree with us using `omega` from now on, but I don't think we should waster time changing that within this PR